flink-runtime/src/test/java/org/apache/flink/runtime/util/TestingTaskManagerRuntimeInfo.java

```diff
@@ -38,7 +38,8 @@
     public TestingTaskManagerRuntimeInfo() {
         this(
                 new Configuration(),
-                System.getProperty("java.io.tmpdir").split(",|" + File.pathSeparator));
+                EnvironmentInformation.getTemporaryFileDirectory()
```

Comment:
We should use junit temp dir here instead, but that's ok for now if it fixes the issue

Reply:
I thought about it, but this is a helper class, it's not a "junit" specific class, and I thought we shouldn't use `TemporaryFolder` directly.
Another idea is to remove ctors with no `tmpWorkingDir` so it's a mandatory argument, and all consumers should be responsible to pass a directory path there, so all the tests using it could pass a junit `TemporaryFolder.newFolder()`.

Reply:
@Myasuka what do you think ^^ ?